pkg/security/probe/model.go

```diff
@@ -209,20 +208,19 @@ func (ev *Event) ResolveContainerTags(e *model.ContainerContext) []string {
 	return e.Tags
 }
 
-// UnmarshalProcess unmarshal a Process
-func (ev *Event) UnmarshalProcess(data []byte) (int, error) {
+// UnmarshalProcessCacheEntry unmarshal a Process
+func (ev *Event) UnmarshalProcessCacheEntry(data []byte) (int, error) {
 	// reset the process cache entry of the current event
 	entry := ev.resolvers.ProcessResolver.NewProcessCacheEntry()
-	entry.Pid = ev.ProcessContext.Pid
-	entry.Tid = ev.ProcessContext.Tid
+	entry.PIDContext = ev.PIDContext
 
 	n, err := entry.Process.UnmarshalBinary(data)
 	if err != nil {
```

Comment:
We should release the process cache entry here

Reply:
It is not Retained yet so no need to release it

Reply:
It is not Retained yet so no need to release it